value-processor/src/org/immutables/value/processor/Immutables.generator

```diff
@@ -1307,11 +1309,27 @@ public interface BuildFinal[type.generics] {
   [atCanIgnoreReturnValue type]
   public final [builderReturnType type] [type.names.from]([type.typeAbstract.relative] instance) {
     [requireNonNull type](instance, "instance");
+    [dynamicFromModifiableCheck type 'instance' 'this']
     [for v in setters]
     [buildFromAttribute v]
     [/for]
     return [builderReturnThis type];
   }
+  [/if]
+  [if type.kind.isModifiable]
+  /**
```

Comment:
probably need to add a blank line inside `[if` before the method. For methods, nested types and sometimes group of fields, when we have conditional output we put separating blank lines inside `[if` or `[for` directives, but before the methods so generated code would have needed blank lines and will not contain extra blank lines if optional code generated and when not.